src/wasm/wasm-type.cpp

```diff
@@ -42,8 +128,18 @@ const char* printType(Type type) {
       return "exnref";
     case Type::unreachable:
       return "unreachable";
+    default: {
+      std::vector<Type::ValueType> vts = type.getValueTypes();
+      std::string res("(");
+      for (size_t i = 0; i < vts.size() - 1; ++i) {
+        res += ", ";
```

Comment:
Doesn't this print something like `(, type1, type2, ...)`?

Reply:
Yes, oops.